pkg/ipam/prefix/prefix.go

```diff
@@ -101,11 +101,11 @@ type listResponse struct {
 	}
 }
 
-func (a api) List(ctx context.Context, page, limit int) ([]Summary, error) {
+func (a api) List(ctx context.Context, page, limit int, search string) ([]Summary, error) {
 	url := fmt.Sprintf(
-		"%s%s?page=%v&limit=%v",
+		"%s%s?page=%v&limit=%v&search=%s",
```

Comment:
oh welp, that would allow me to do any kind of whacky shit :D

example: I can set `search` to `"&any_parameter_i_want=foo"` and that would get passed as is. Would love it this would get refactored to use `url.Values`, that way spaces and such would get properly URL encoded as well. :3